sark/code/function.py

```diff
@@ -181,6 +181,14 @@ def crefs_to(self):
         return idautils.CodeRefsTo(self.startEA, 1)
 
     @property
+    def calls(self):
+        """
+        Calls to other functions.
+        Returns: Xref objects
+        """
+        return ifilter(lambda x: x.type.is_call, self.xrefs_from)
```

Comment:
Since you have to define a `lambda`, `(xref for xref in self.xrefs_from if xref.type.is_call)` seems the more "Pythonic" solution.